shadow-cljs.edn

```diff
@@ -82,6 +82,8 @@
            :output-dir       "target/test"
            :optimizations    :simple
            :target           :node-test
+           ;; When running tests without a REPL you can uncomment the below line to `make test-watch` a specific file
```

Comment:
can this just be passed in as a parameter to the `make test-watch` command ??

Reply:
Yes, one way is via a environment variable, because ShadowCLJS supports this in the `edn` file.

So for example, something like this should be doable:

```bash
NS_REGEX='status-im.chat.models-test$' make test-watch
```